prisma/seed.ts

```diff
@@ -103,7 +99,7 @@ async function main() {
     });
     const userBuyQuantity = faker.datatype.number({ min: 1, max: 10 });
     const userBuyOfferCreatedAt = faker.date.recent();
-    const userBuyOfferStatus = faker.datatype.number(statusCountBuyOffer);
+    const userBuyOfferStatus :OfferStatus = "ACTIVE";
```

Comment:
I'm pretty sure you can also do it like this. It doesn't need to be changed. Just a tip.
```TS
const userBuyOfferStatus = OfferStatus.ACTIVE;
```